src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -13,49 +13,26 @@ import {
 } from '../../../../common/util/util.js';
 import { GPUTest } from '../../../gpu_test.js';
 
-export const g = makeTestGroup(GPUTest);
+class DrawTest extends GPUTest {
+  checkTriangleDraw(
+    first_index: number | undefined,
+    count: number,
+    first_instance: number | undefined,
+    instance_count: number | undefined,
+    indexed: boolean,
+    indirect: boolean,
+    vertex_buffer_offset: number,
+    index_buffer_offset: number | undefined,
+    base_vertex: number | undefined
```

Comment:
nit: For readability, prefer to use "named" arguments by doing something like:

```ts
  checkTriangleDraw(opts: {
    first_index: number | undefined,
    count: number,
    first_instance: number | undefined,
    instance_count: number | undefined,
    indexed: boolean,
    indirect: boolean,
    vertex_buffer_offset: number,
    index_buffer_offset: number | undefined,
    base_vertex: number | undefined
  }): void {
```

Reply:
Done.